jest.conf/visual.testUtils.js

```diff
@@ -40,8 +41,30 @@ export async function renderComponent(component, props) {
   global.expect(isComponentRendered).toBe(true);
 }
 
-export async function takeSnapshot(name) {
+export async function takeSnapshot(name, options = {}) {
```

Comment:
Here would be good also to have some jsdoc, and let users know what options could we use (I think it should be a lot of options that percySnapshot receives, in any case we could link to their documentation).

Reply:
Sure 👍